### PR TITLE
Docker image update for k8 event module

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

File: images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml [hunk 1]
```diff
@@ -1,14 +1,35 @@
 ---
 
-- name: Print meteringconfig values
+- name: run the new module
+  k8s_event:
+    state: present
+    name: test-https-emily109
+    namespace: ansible-emily
```
Comment (timflannagan1) [inline images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml]: `namespace: {{ meta.namespace }}`

File: images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
-    msg: "{{ item }}: {{ lookup('vars', item) }}"
-  loop: "{{ vars | dict2items | to_json | from_json | json_query(query) | difference(filtered_vars) }}"
-  vars:
-    filtered_vars:
-      - meteringconfig_spec
-      - meteringconfig_spec_overrides
-      - meteringconfig_default_values
-    query: "[?starts_with(@.key, `meteringconfig_`)].key"
+    msg: '{{ testout }}'
+# - name: Print meteringconfig values
```
Comment (timflannagan1) [inline images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml]: uncomment the rest of the main.yml tasks

File: images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml (hunk 1 above)
Comment (EmilyM1) [reply images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml]: I threw the WIP on it. The main.yml is not being commited. For test only. Now reverted.

Comment (timflannagan1) [reply images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml]: I don't see a problem with committing this to main.yml for the time being. If you wanted to just test the event in the e2e suite, you could have run this locally.

Comment (timflannagan1) [reply images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml]: This wouldn't pass the e2e test either as the `ansible-emily` namespace doesn't exist in CI.

**Comment by timflannagan1** [comment]
I think we may need to also update this:
https://github.com/operator-framework/operator-metering/blob/d7e7ff5be86b523bbc7beda9c463dc728a6605e0/images/metering-ansible-operator/roles/meteringconfig/library/k8s_event.py#L204

File: images/metering-ansible-operator/roles/meteringconfig/library/k8s_event.py [hunk 3]
```diff
@@ -199,9 +200,9 @@
        returned: success
        type: str
      source:
-       description: Component for reporting this Event
+       description: Component for reporting this Event. Includes source.
```
Comment (timflannagan1) [inline images/metering-ansible-operator/roles/meteringconfig/library/k8s_event.py]: nit: Let's mirror the description of [`corev1.EventSource`](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L5057).

Comment (timflannagan1) [reply images/metering-ansible-operator/roles/meteringconfig/library/k8s_event.py]: `description: source contains information for an event.`

**Pull request merged**